app/models/programme.rb

```diff
@@ -60,6 +60,17 @@ def enough_activities_for_test?(_user)
     false
   end
 
+  def show_notification_for_test?(user)
+    false
+  end
+
+  def in_notifiable_period?(user)
+    enrolment = user.user_programme_enrolments.find_by(programme_id: id)
+    notifiable_date_range = 48.hours.ago..Time.current
+
+    notifiable_date_range.cover?(enrolment.last_enrolment_date)
+  end
```

Comment:
This method feels like an implementation detail that is only relevant for CSA. Could this be moved onto the CSA model?